code/def_files/data/effects/main-f.sdr

```diff
@@ -293,10 +293,17 @@ void main()
  #ifdef FLAG_TEAMCOLOR
 	vec4 teamMask = vec4(0.0, 0.0, 0.0, 0.0);
 	teamMask = texture(sMiscmap, vec3(texCoord, float(sMiscmapIndex)));
-	vec3 base = max(base_color - vec3(0.5), vec3(0.0));
-	vec3 stripe = max(stripe_color - vec3(0.5), vec3(0.0));
-	baseColor.rgb += (base * teamMask.x) + (stripe * teamMask.y);
+	vec3 team_color = (base_color * teamMask.x) + (stripe_color * teamMask.y);
```

Comment:
`base_color` and `stripe_color` are used without changes here but in the previous version `0.5` was subtracted from it. Could you explain why you made this change?

Reply:
That was actually an oversight. I was experimenting around with it and this change snuck in by accident.